.github/scripts/check_icon_pr.py

```diff
@@ -41,7 +46,7 @@ def main():
 
         err_msg = []
         if devicon_err_msg != "":
-            err_msg.append(devicon_err_msg)
+            err_msg += devicon_err_msg
```

Comment:
Whoopsie. This should probably use .append like the others :)

Reply:
nvm, .append is for strings only, so we can use `+=` or `.extend`, of which I prefer `.extend`
```suggestion
            err_ms.extend(devicon_err_msg)
```